tsl/src/continuous_aggs/materialize.c

```diff
@@ -620,12 +649,33 @@ materialization_invalidation_log_get_range(int32 materialization_id, Oid type, i
 		if (entry_range.start >= completed_threshold)
```

Comment:
should we also have `if (entry_range.start >= invalidate_prior_to_time) break` ?